### ISSUE TITLE
No notification when client is not running due to modal dialog

### ISSUE THREAD

**Issue by mehturt** [vector-im/riot-android#2306]
Not sure if "modal dialog" is the correct wording - the issue is that suppose application is upgraded and there is a notification dialog that must be accepted by user before application connects.  If the application is upgraded by Android automatically, the user is not aware of this, and actually is not receiving any notifications about incoming messages (because client is not connected, so it's obvious).
Could there be a notification that user has to do something in order for application to connect?
I think similar issue is when server certificate changes, user must accept it before client connects, but again may not be aware of this if it does not run application by himself.

**Comment by bmarty** [comment]
Hi @ndarilek ,

Thanks for reporting this issue, maybe related to GDPR consent dialog.

Benoît

**Comment by mehturt** [comment]
Yes I guess last time it was after the GDPR upgrade, but as I said in the original report, it may as well be during server certificate change.
